Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -201,15 +202,18 @@ class AttachmentSaveEventListener final : public EventListener {
     void handleEvent(ScriptExecutionContext&, Event& event) final
     {
         if (event.type() == eventNames().clickEvent) {
+            ASSERT(event.isMouseEvent());
+            auto& mouse = static_cast<MouseEvent&>(event);
+            auto copiedEvent = MouseEvent::create(m_attachment->attributeWithoutSynchronization(saveAttr()), mouse.bubbles() ? Event::CanBubble::Yes : Event::CanBubble::No,
+                mouse.cancelable() ? Event::IsCancelable::Yes : Event::IsCancelable::No, mouse.composed() ? Event::IsComposed::Yes : Event::IsComposed::No,
```

Comment:
Afaik the event is not cancelable, `event.preventDefault()` from the save event wouldn't do anything here.